pokerole_irl/user_app/views.py

```diff
@@ -42,4 +43,57 @@ def dispatch(self, request, *args, **kwargs):
     form_class = CreateUserForm
     template_name = 'registration/register.html'
     success_url = '/login'
-    model = get_user_model()
\ No newline at end of file
+    model = get_user_model()
+
+class SettingListView(LoginRequiredMixin, ListView):
+    template_name = "users/setting_list.html"
+    model = Setting
+    context_object_name = "settings"
+    paginate_by = 20
+
+    def get_queryset(self):
+        queryset = super().get_queryset()
+        return queryset.filter(members=self.request.user)
```

Comment:
You need to user member__contains here I think

Reply:
Not quite sure what that does. Can you explain?